Classes/Explorer Toolbar/FLEXManager.h

```diff
@@ -23,9 +23,18 @@
 /// @param entryName The string to be displayed in the cell.
 /// @param objectFutureBlock When you tap on the row, information about the object returned by this block will be displayed.
 /// Passing a block that returns an object allows you to display information about an object whose actual pointer may change at runtime (e.g. +currentUser)
+/// @param forceObjectExplorer Specifies how to interpret object returned from objectFutureBlock. If YES it will create object/class inspector view controller
+/// for any supplied object, while NO will cause any passed UIViewController subclasses to be pushed into manager's navigation view controller.
 /// @note This method must be called from the main thread.
 /// The objectFutureBlock will be invoked from the main thread and may return nil.
 /// @note The passed block will be copied and retain for the duration of the application, you may want to use __weak references.
+- (void)registerGlobalEntryWithName:(NSString *)entryName
```

Comment:
What do you think about separating the two API a little? I think something like:

```objc
- (void)registerGlobalEntryWithName:(NSString *)entryName viewControllerFuture:(UIViewController (^)(void))viewControllerFuture;
```

would be a little more clear than the force `BOOL`. The implementation would also be pretty simple since `FLEXGlobalsTableViewControllerEntry` already consists of a name future and a view controller future.